app/js/components/orderEntry/Paginate.jsx

```diff
@@ -2,6 +2,21 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 
+const getNextPageNumber = (nextPageUrl, totalNumberOfOrders) => {
+  const maxNumberOfOrdersOnAPage = 10;
+  if (nextPageUrl) {
+    const pageNumber = Math.floor(nextPageUrl.split('=')[nextPageUrl.split('=').length - 1] / maxNumberOfOrdersOnAPage); 
+    return pageNumber;
+  }
+  return Math.ceil(totalNumberOfOrders / maxNumberOfOrdersOnAPage);
+};
+
+const getNumberOfAllPages = (totalNumberOfOrders) => {
+  const maxNumberOfOrdersOnAPage = 10;
```

Comment:
Why are you assigning maxNumberOfOrdersOnAPage more than once?

Reply:
They are assigned scoped to a function.

Reply:
So if we change the number from 10 to some thing else, does this mean we have to change it in two places?